OpenSim/Common/TimeSeriesTable.h

```diff
@@ -129,6 +129,16 @@ class InvalidTimeRange : public Exception {
     }
 };
 
+class InvalidIndices : public Exception {
+public:
+    InvalidIndices(const std::string& file,
+                   size_t line,
+                   const std::string& func) :
+        Exception(file, line, func) {
+        addMessage("last_index is out of bounds.");
```

Comment:
Consider being more descriptive in this error message: "Index `index` is out of bounds [`min`, `max`]". Use arguments to the custom exception to fill in `index`, `min`, and `max`, similar to the exceptions above.